script/tasks/diamondUpdateFacet.sh

```
@@ -140,10 +147,17 @@
         CLEAN_RETURN_DATA=$(echo $RAW_RETURN_DATA | sed 's/^.*{\"logs/{\"logs/')
         FACET_CUT=$(echo $CLEAN_RETURN_DATA | jq -r '.returns.cutData.value')
-        if [ "$FACET_CUT" != "0x" ]; then
-          echo "Proposing facet cut for $script on network $NETWROK..."
-          DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
-          npx tsx script/deploy/safe/propose-to-safe.ts --to "$DIAMOND_ADDRESS" --calldata "$FACET_CUT" --network "$NETWORK" --rpcUrl $(getRPCUrl $NETWORK) --privateKey "$SAFE_SIGNER_PRIVATE_KEY"
+        if [ "$FACET_CUT" == "0x" ] || [ -z "$FACET_CUT" ]; then
+          error "Unable to extract facet cut data from RPC response at logs.returns.cutData.value"
+        else
+          if [ "$DEPLOY_NEW_NETWORK_MODE" == "true" ]; then
+            echo "DEPLOY_NEW_NETWORK_MODE is activated - executing facet cut for $script on network $NETWORK..."
+            RAW_RETURN_DATA=$(NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND NO_BROADCAST=false PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") forge script "$SCRIPT_PATH" -f $NETWORK -vvvvv --json --broadcast --skip-simulation --legacy)
```

Shellcheck (0.10.0)

[warning] This assignment is only seen by the forked process.

(SC2097)